hack/windows/archive-proxy/main.go

```diff
@@ -0,0 +1,94 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/elastic/elastic-agent/pkg/core/process"
+)
+
+var CommitSHA string
```

Comment:
Worth adding a comment that this is set by the linker at build time.

Reply:
I am always up for improving readability, thanks for the point 🙂 I added it here bc4c38c8b1f51e1d7e79d16008e8ea6497daa2da, please feel free to unresolve if you spot anything else